backend/src/tasks/test/notifications.test.ts

```diff
@@ -0,0 +1,223 @@
+    const firstName = 'first names';
+    const lastName = 'last names';
+    const email = Math.random() + '@crossfeed.cisa.gov';
+    const user = await User.create({
+      firstName: '',
+      lastName: '',
+      email: Math.random() + '@crossfeed.cisa.gov'
+    }).save();
+    const user2 = await User.create({
+      firstName: '',
+      lastName: '',
+      email: Math.random() + '@crossfeed.cisa.gov'
+    }).save();
+    const user3 = await User.create({
+      firstName: '',
+      lastName: '',
+      email: Math.random() + '@crossfeed.cisa.gov'
+    }).save();
+    await Role.create({
+      role: 'admin',
+      approved: false,
+      organization: organization,
+      user: user
+    }).save();
+    await Role.create({
+      role: 'admin',
+      approved: false,
+      organization: organization2,
+      user: user2
+    }).save();
+    await Role.create({
+      role: 'user',
+      approved: false,
+      organization: organization3,
+      user: user3
+    }).save();
+
+    const response = await request(app)
+      .post(`/users`)
+      .set(
+        'Authorization',
+        createUserToken({
+          roles: [{ org: organization.id, role: 'admin' }]
+        })
+      )
+      .send({
+        firstName,
+        lastName,
+        email,
+        organization: organization.id,
+        organizationAdmin: true
+      })
+      .expect(200);
+    expect(response.body.roles[0].role).toEqual('admin');
+  });
+
+  test('getting reports list for all organizations', async () => {
+    const user = await User.create({
+      firstName: 'Dabi',
+      lastName: '',
+      email: Math.random() + '@crossfeed.cisa.gov'
+    }).save();
+    await Role.create({
+      role: 'user',
+      approved: false,
+      organization,
+      user
+    }).save();
+
+    const response = await request(app)
+      .post('/reports/list')
+      .set(
+        'Authorization',
+        createUserToken({
+          roles: [{ org: organization.id, role: 'user' }]
+        })
+      )
+      .send({ currentOrganization: { id: organization.id } })
+      .expect(200);
+    expect(response.text).toEqual('{"Contents":"report content"}');
+    expect(listReports).toBeCalled;
+  });
+  test('Emails to regular user should not send', async () => {
+    const firstName = 'first names';
+    const lastName = 'last names';
+    const email = Math.random() + '@crossfeed.cisa.gov';
+    const user = await User.create({
+      firstName,
+      lastName,
+      email
+    }).save();
+
+    const response = await request(app)
+      .post('/users')
+      .set(
+        'Authorization',
+        createUserToken({
+          roles: [{ org: organization.id, role: 'user' }]
+        })
+      )
+      .send({
+        firstName,
+        lastName,
+        email,
+        organization: organization.id,
+        organizationAdmin: false
+      })
+      .expect(403);
+  });
+  test('Emails to org admin user should send', async () => {
```

Comment:
```suggestion
  test('Notification emails should send to admins', async () => {
```

Reply:
updated in [967bf0e](https://github.com/cisagov/crossfeed/pull/1709/commits/967bf0e8bd390b5560e8381d590f64a4ec528a16)